src/0.9.1/command-reference.md

```diff
@@ -0,0 +1,247 @@
+# Command reference
+
+## immudb
+
+Simply run ```./immudb -d``` to start immudb locally in the background.
+
+If you want to stop immudb în that case you need to find the process `ps -ax | grep immudb` and then `kill -15 <pid>`. Windows PowerShell would be `Get-Process immudb* | Stop-Process`.
```

Comment:
I really like the fact that the normal stop is explicitly mentioned here.
It would be nice also to specify it in the Quickstart page(s), in _Run immudb binary_ section.